packages/frontend/src/components/Settings/Advanced.tsx

```diff
@@ -53,3 +60,40 @@ export default function Advanced({ settingsStore }: Props) {
     </>
   )
 }
+
+function SettingsAutoStart() {
+  const tx = useTranslationFunction()
+
+  const [autostartState, setAutostartState] = useState<AutostartState | null>(
+    null
+  )
+  const update = useCallback(() => {
+    runtime.getAutostartState().then(setAutostartState)
+  }, [])
+
+  useEffect(() => {
+    update()
+  }, [update])
+
+  return (
+    <>
+      <DesktopSettingsSwitch
+        settingsKey='autostart'
+        label={tx('pref_autostart')}
+        description={
+          autostartState
+            ? autostartState.isRegistered
+              ? tx('pref_autostart_registered')
+              : tx('pref_autostart_not_registered')
```

Comment:
Also perhaps there is no need to display these, unless we failed to register / unregister autostart. The value of the toggle should be enough.
But this is also not too important.

Reply:
> unless we failed to register / unregister autostart

too much complexity for now, we can always improve